### PR TITLE
Update to esConsumes and ready for processing Run3 data

### PR DESCRIPTION
**Pull request by PascalBaertschi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ Prepare your working directory with CMSSW
 
 ```
 export SCRAM_ARCH=slc7_amd64_gcc820
-cmsrel CMSSW_11_1_4
-cd CMSSW_11_1_4/src
+cmsrel CMSSW_12_1_1
+cd CMSSW_12_1_1/src
 cmsenv
-git clone https://github.com/CMSTrackerDPG/SiPixelTools-PixelTriplets.git SiPixelTools/PixelTriplets
+git clone https://github.com/PascalBaertschi/SiPixelTools-PixelTriplets.git SiPixelTools/PixelTriplets
```
Comment (ferencek) [inline README.md]:
```suggestion
git clone https://github.com/CMSTrackerDPG/SiPixelTools-PixelTriplets.git SiPixelTools/PixelTriplets
```

File: README.md [hunk 2]
```diff
@@ -9,13 +9,14 @@ Prepare your working directory with CMSSW
 
 ```
 export SCRAM_ARCH=slc7_amd64_gcc820
```
Comment (ferencek) [inline README.md]:
```suggestion
export SCRAM_ARCH=slc7_amd64_gcc900
```

**Comment by ferencek** [comment]
@PascalBaertschi, once you accept an invitation to the SiPixelTools team, you will be able to push directly to the main repo (this was the original intention of this team as described in https://hypernews.cern.ch/HyperNews/CMS/get/pixelOfflineSW/1556/1.html). For future developments you can either continue creating PRs or can push directly to a branch of interest (the latter makes sense in particular if you are the only active user of the package). Development branches from which you can create a PR can also be pushed to the main repo.

**Comment by PascalBaertschi** [comment]
@ferencek Thanks for adding me to the SiPixelTools team. I will check what works better for me, directly push to the repo or first to my fork and then making a PR.

**Pull request merged**